### ISSUE TITLE
Relationship Field sort_field must be unique

### ISSUE THREAD

**Issue by gabrielfreiberg** [FrozenNode/Laravel-Administrator#947]
The [*belongsToMany* Field Type Relationship](http://administrator.frozennode.com/docs/field-type-relationship#belongs-to-many) get's confused if you define a *sort_field* in a pivot table with the same name as a column in one of the Model's tables.

**So for example:**
*Schemas:*
projects(id, title, body, image, **position**)
tags(id, title)
project_tag(id,  **position**)

*Models:*
Tag belongsToMany Project
Project belongsToMany Tag

*Administrator model config file:*
	'edit_fields' => array(
		'projects' => array(
			'title' => 'Projects',
			'type' => 'relationship',
			'name_field' => 'title',
			'sort_field' => 'position'
		),
	),

This will cause the sorting to not work due to the fact that the "position" in the pivot table is the same name as "position" in the Project table.

**Comment by davemathews** [comment]
I would need to see the full admin configs and your models, I am not sure where the problem lies with what you have given.

**Comment by gabrielfreiberg** [comment]
Sure thing, I'm happy to help reproduce the issue. What format would be most convenient for you?

Would a skeleton structure of Models, Migrations, and Config files (same as the /examples directory) suffice?

**Comment by davemathews** [comment]
just post here the raw code and I will take a look.  give me the 2 models, and the admin config giving you the issue.

**Comment by davemathews** [comment]
any movement on this?

**Comment by gabrielfreiberg** [comment]
yes, please find the models and config required to reproduce the issue attached
[Laravel-Administrator-issue-947.zip](https://github.com/FrozenNode/Laravel-Administrator/files/74483/Laravel-Administrator-issue-947.zip)

**Comment by gabrielfreiberg** [comment]
if this is a confirmed bug and you're accepting pull requests, i'd be happy to try and provide a fix

**Comment by gabrielfreiberg** [comment]
this issue is still labeled as not having enough information, let me know if you need anything additional
